code/datums/station_traits/neutral_traits.dm

```diff
@@ -219,11 +221,12 @@
 			message_admins("Attempted to make [birthday_override_ckey] the birthday person but they are not a valid station role. A random birthday person has be selected instead.")		
 
 	if(!birthday_person)
-		var/list/birthday_options
+		var/list/birthday_options = list()
 		for(var/mob/living/carbon/human/human in GLOB.human_list)
 			if(human.mind?.assigned_role.job_flags & JOB_CREW_MEMBER)
-				birthday_options += human
+				birthday_options.Add(human)
```

Comment:
why

Reply:
Jacquerel got there before you, I've changed it back to +=